build.gradle

```diff
@@ -1,159 +1,113 @@
-plugins {
-    id 'com.android.application' version '7.3.1'
-    id 'com.android.library' version '7.3.1' apply false
+ext {
+    assetsPath = "${rootProject.projectDir}/assets"
+    thirdPartyAssetsPath = "${rootProject.projectDir}/third_party/assets"
 }
 
-android {
-    namespace 'com.google.bigwheels'
-    compileSdk 32
+subprojects {
+    repositories {
+        mavenCentral()
+        google()
+    }
 
-    defaultConfig {
-        applicationId "com.google.bigwheels"
-        minSdk 29
-        targetSdk 32
-        versionCode 1
-        versionName "1.0"
+    apply plugin: "com.android.application"
```

Comment:
missed it before: no idea if that's important, but before we fixed the application plugin version to something. Seems like it would be a good thing to keep?

Reply:
Hm, I thought the version came from `com.android.tools.build:gradle` in the `settings.gradle` but can't find a confirmation on that. I added a plugins block with version to `settings.gradle` for this.